app.rambox.ramboxce.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: app.rambox.ramboxce
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+add-extensions:
+  org.freedesktop.Platform.GL:
+    directory: lib/linux-gnu/GL
+    version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=xdg-config/kdeglobals:ro
+  - --filesystem=xdg-download:create
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.AppMenu.Registrar.*
+command: ramboxce
+tags: [opensource]
+separate-locales: false
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - name: ramboxce
+    buildsystem: simple
+    build-commands:
+      - install -d /app/share/ramboxce
+      - install -Dm755 ramboxce.sh /app/bin/ramboxce
+      - install -Dm644 app.rambox.ramboxce.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 app.rambox.ramboxce.desktop -t /app/share/applications
```

Comment:
The .deb being used here already has `/usr/share/applications/rambox.desktop`, just `mv` it to `/app/share/applications` and at top-level `rename-desktop-file: rambox.desktop`